src/ios/CodePush.m

```diff
@@ -314,6 +314,10 @@ - (void)saveFailedUpdate:(NSString *)packageHash {
     NSMutableArray* failedUpdates = [preferences objectForKey:FailedUpdatesKey];
```

Comment:
It works fine, I triple checked it. `mutableCopy` works on any `NSObject`, and for `nil` it just returns `nil` again.